### PR TITLE
[DOCS] Update Cloud Docs to Include Docker Methodology

### PR DESCRIPTION
**Pull request by kwcanuck**
In [DSB-365](https://greatexpectations.atlassian.net/browse/DSB-365), @allensallinger requested that the existing GX Cloud documentation be revised to include the Docker methodology for setting the environment variables and installing and running the GX Cloud Agent. This PR implements these updates. 

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] _Appropriate_ tests and docs have been updated

[DSB-365]: https://greatexpectations.atlassian.net/browse/DSB-365?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To learn more about Data Assets, see [Data Asset](../../terms/data_asset.md).
 
 - You have [set up your environment](../set_up_gx_cloud.md) including setting the `GX_CLOUD_SNOWFLAKE_PASSWORD` environment variable. 
 
-- You have a [Snowflake account](https://docs.snowflake.com/en/user-guide-admin) with `ACCOUNTADMIN` access.
+- You have a [Snowflake account](https://docs.snowflake.com/en/user-guide-admin) with `ACCOUNTADMIN` access and know the password.
```
Comment (allensallinger) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: I don't think that the snowflake account needs to have `ACCOUNTADMIN` access. The user needs access to read and from the specified DB/SCHEMA/TABLE.

Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @allensallinger, I have updated the text. Please review and provide your edits or approval. Thanks!

**Review by allensallinger — APPROVED** [review]

**Review by allensallinger — APPROVED** [review]
LGTM

**Comment by erik-hencier** [comment]
looks good to me flow-wise, Allen knows the technical correctness

**Review by talagluck — APPROVED** [review]
LGTM

**Pull request merged**